### PR TITLE
Text changed for adding first contact

### PR DESCRIPTION
**Pull request by webtaculars**
When a user gets signed up, he hasn't added any contacts. Still he is shown "Add or edit a contact". 
I have changed the text of the adding the first contact button to "Add contact". 
Please review the PR @djaiss

### CODE REVIEW

File: resources/lang/fr/dashboard.php [hunk 1]
```diff
@@ -16,7 +16,7 @@
     'event_activity_about' => 'une activité à propos de',
     'event_tasks_about' => 'une tâche concernant',
     'blank_title' => 'Vous n\'avez encore aucune activité.',
-    'blank_cta' => 'Ajoutez ou éditez un contact',
+    'blank_cta' => 'Ajoutez contact',
```
Comment (djaiss) [inline resources/lang/fr/dashboard.php]: It would be `Ajouter un contact` there.

**Comment by djaiss** [comment]
Just added one comment. Can you review? Thanks a lot!

**Comment by webtaculars** [comment]
I will surely update it. I got a passing grade in French in college :)

**Comment by webtaculars** [comment]
@djaiss Fixed it

**Comment by djaiss** [comment]
Thank so much for caring about the project and for this pull request.
The PR looks awesome, I'll merge it right away.

**Pull request merged**